lib/utils.js

```diff
@@ -9,128 +9,136 @@ const parseJson = require('parse-json');
 const { hasher } = require('@asset-pipe/common');
 const { default: Worker } = require('jest-worker');
 
-const bundleWorker = new Worker(require.resolve('./bundler'), {
-    numWorkers: 2,
-});
+module.exports = ({ bundleInProcess = false } = {}) => {
+    let bundler;
 
-async function fetchFeeds(sink, ids) {
-    try {
-        return await Promise.all(
-            ids.map(async fileName => ({
-                fileName,
-                contents: await sink.get(fileName),
-            }))
-        );
-    } catch (err) {
-        throw Boom.boomify(err, {
-            message: 'Unable to fetch 1 or more feeds.',
+    if (bundleInProcess) {
```

Comment:
I think we should just move bundling out of `utils`.

Reply:
works for me